iroh-resolver/src/unixfs_builder.rs

```diff
@@ -561,21 +584,88 @@ impl Store for Arc<tokio::sync::Mutex<std::collections::HashMap<Cid, Bytes>>> {
     }
 }
 
+/// Configuration for adding unixfs content
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Config {
+    /// Should the outer object be wrapped in a directory?
+    pub wrap: bool,
+    pub chunker: ChunkerConfig,
+}
+
+/// Chunker configuration.
+#[derive(Debug, Clone, PartialEq, Eq, Copy)]
+pub enum ChunkerConfig {
+    /// Fixed sized chunker.
+    Fixed(usize),
+    /// Rabin chunker.
+    Rabin,
```

Comment:
I had expected the rabin chunker to have lots of options. I guess we go with reasonable defaults for now...

Reply:
honestly, I have no idea which of the settings to expose, so I went with none for now